Engines/Wine/Engine/Implementation/script.js

```diff
@@ -23,6 +23,7 @@ module.default = class WineEngine {
         this._ldPath = propertyReader.getProperty("application.environment.ld");
         this._wineEnginesDirectory = propertyReader.getProperty("application.user.engines") + "/wine";
         this._winePrefixesDirectory = propertyReader.getProperty("application.user.containers") + "/" + WINE_PREFIX_DIR + "/";
+        this._useRuntime = (propertyReader.getProperty("application.environment.wineRuntime") !== "false");
```

Comment:
The braces are not required

Reply:
I know. I've put them only to make it more clear.